### ISSUE TITLE
'DEFAULT_ANKI_CLOZE' is not defined

### ISSUE THREAD

**Issue by khonkhortisan** [ssricardo/anki-plugins#27]
```
Error 
Se ha producido un error. Por favor inicia la Anki manteniendo presionadas ples teclas Mayús y la flecha hacia abajo, lo que desactivará temporalmente los extensiones que tienes instaladas. 
Si este error persiste sólo cuando tienes las extensiones activadas, utiliza Herramientas > Menú de Extensiones para deshabilitar alguna extensión y reinicia el Anki, repitiendo este proceso hasta que descubras qué extensión en concreto es la que causa el problema 
Cuando hayas descubierto la extensión que causaba el problema, por favor informa de esto a sección de extensiones de la nuestra web de apoyo. 
Información de depuración: 
Anki 2.1.14 (7b93e985) Python 3.6.7 Qt 5.12.1 PyQt 5.11.3
Platform: Windows 10
Flags: frz=True ao=True sv=2

Caught exception:
  File "aqt\webview.py", line 322, in handler
  File "aqt\reviewer.py", line 281, in _onAnswerButton
  File "<decorator-gen-152>", line 2, in _answerCard
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 71, in repl
  File "C:\Users\Khonkhortisan\AppData\Roaming\Anki2\addons21\935264945\main.py", line 214, in answerCard
    _old(self, ease)
  File "<decorator-gen-148>", line 2, in _answerCard
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 69, in repl
  File "<decorator-gen-22>", line 2, in _answerCard
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 71, in repl
  File "C:\Users\Khonkhortisan\AppData\Roaming\Anki2\addons21\1212877655\__init__.py", line 18, in newAnswerCard
    _old(self, min(self.mw.col.sched.answerButtons(self.card), ease))
  File "<decorator-gen-4>", line 2, in _answerCard
  File "anki\hooks.py", line 74, in decorator_wrapper
  File "anki\hooks.py", line 69, in repl
  File "aqt\reviewer.py", line 240, in _answerCard
  File "aqt\reviewer.py", line 96, in nextCard
  File "aqt\reviewer.py", line 172, in _showQuestion
  File "aqt\reviewer.py", line 156, in _mungeQA
  File "aqt\reviewer.py", line 304, in typeAnsFilter
  File "C:\Users\Khonkhortisan\AppData\Roaming\Anki2\addons21\1933645497\handler.py", line 76, in typeAnsQuestionFilter
    return re.sub(ref.typeAnsPat, DEFAULT_ANKI_CLOZE % (ref.typeFont, ref.typeSize), buf)
<class 'NameError'>: name 'DEFAULT_ANKI_CLOZE' is not defined
```
This is an odd error. It goes through Incremental Reading v4.10.3, Right Hand Reviews (jkl;), Fill the blanks - Multiple type:cloze support.
I answer one card, it throws the error for the next card but doesn't show me the question or update the answer buttons, then when I press space it puts the answer on the same line as the question. 
Note type: Type
Card: Tarjeta 1
Fields: Front, Type, Extra
Front template:
```
{{Front}}
{{type:Type}}
```
Back template:
```
{{FrontSide}}

<hr id=answer>

{{Extra}}
```
It also doesn't let me escape the template editor.

**Comment by galantra** [comment]
Same thing occurs to me sometimes.

**Comment by ssricardo** [comment]
Hi! Thanks for the detailed report. 
But indeed an odd behavior, not that easy to find out the problem. 

Did you try to deactivate these other addons, just to check if that still happens?

**Comment by MikeMoolenaar** [comment]
Also have this problem. Tried disabling other addons, but no luck.
I did fix it by changing `DEFAULT_ANKI_CLOZE` to `self.DEFAULT_ANKI_CLOZE` in handler.py on line 76.

**Comment by ssricardo** [comment]
Thanks for the tip @MikeMoolenaar .

I think what is happening is the this function is wrapped and called from somewhere else, then it couldn't find the variable in a different context.

**Comment by ssricardo** [comment]
Released version 1.4 to handle this.
Let me know if the error still happens

**Comment by ssricardo** [comment]
Closing with version 1.4 fix. 
Please, reopen if it persists
